pkg/nad/masterplugin.go

```diff
@@ -380,7 +380,7 @@ func (plugin *MasterIPVlanPlugin) WithLinkInContainer() *MasterIPVlanPlugin {
 		return plugin
 	}
 
-	plugin.masterPlugin.LinkInContainer = true
+	plugin.masterPlugin.LinksInContainer = true
```

Comment:
LinkInContainer and LinksInContainer are two different parameters. Please revert it back to it's original state